### PR TITLE
task/3405 add gem cancancan

### PR DESCRIPTION
**Pull request by buitiendo**
redmine: https://edu-redmine.framgia.vn/issues/3405

### CODE REVIEW

**Comment by buitiendo** [comment]
ready

File: app/models/account.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Account < ApplicationRecord
-  devise :database_authenticatable, :registerable,
-   :recoverable, :rememberable
+   devise :database_authenticatable, :registerable,
```
Comment (huuphuc93) [inline app/models/account.rb]: indent 1 space

**Comment by huuphuc93** [comment]
commented

**Comment by buitiendo** [comment]
ready

**Pull request merged**